comms/dht/src/peer_validator.rs

```diff
@@ -98,7 +115,13 @@ impl<'a> PeerValidator<'a> {
             peer.update_addresses(&claim.addresses, &PeerAddressSource::FromDiscovery {
                 peer_identity_claim: claim.clone(),
             });
-            peer.addresses.mark_all_addresses_as_last_seen_now(&claim.addresses);
+            trace!(
+                target: LOG_TARGET,
+                "Peer '{}' / '{}' added with address(es) from claim: {:?}",
+                node_id_hex,
```

Comment:
```suggestion
                node_id,
```

Cloning NodeId (if necessary) is cheap - it should implement the Copy trait but that refactor was never done